tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
+instrument operations in your application.
+Jaeger tracer can be configured with an optional logger that logs errors/spans
+and a stats reporter for emitting metrics.
+Note that the tracer is initialized by default with usage of UberFx modules.
```

Comment:
Yea similar to how logging, metrics etc. is set up. It is possible to configure the uberfx service module with your own custom tracer but that's not relevant in this package doc.